bigbluebutton-html5/imports/api/presentations/server/handlers/presentationExport.js

```diff
@@ -0,0 +1,21 @@
+import { check } from 'meteor/check';
+import sendExportedPresentationChatMsg from '/imports/api/presentations/server/handlers/sendExportedPresentationChatMsg';
+import setPresentationExporting from '/imports/api/presentations/server/modifiers/setPresentationExporting';
+
+export default function handlePresentationExport({ body }, meetingId) {
+  check(body, Object);
+  check(meetingId, String);
+
+  const { fileURI } = body;
+
+  check(fileURI, String);
+
+  const fileURL = new URL(fileURI);
+  const path = fileURL.pathname;
+  const presentationId = path.split('/')[5];
```

Comment:
@danielpetri1, is it possible to include the `presId` within the body of `NewPresAnnFileAvailableMsg` msg? then we can remove this tweak later.

Reply:
It is, should be a small change.

Reply:
Done in the PR linked below